installer/PowerToysSetup/PowerToys.wxs

```diff
@@ -73,6 +73,7 @@
 
         <Chain>
             <ExePackage
+                DisplayName="Exiting PowerToys"
```

Comment:
```suggestion
                DisplayName="Closing PowerToys application"
```

Sounds better to me. Exit sounds not the best in this case because we control other applications. (You can close the dor from outside but you can't leave the house from outside.)

Reply:
Nit. There's also a different way of thinking: "close" sounds like hiding a window, "exit" is clear about the shut-down.
Or: think of it as "PowerToys _is_ exiting" ?